storage/innobase/xtrabackup/utils/debian/rules

```diff
@@ -18,8 +17,8 @@ endif
 override_dh_auto_configure:
 
 ifeq "$(DEB_DUMMY)" ""
-	cmake -DBUILD_CONFIG=xtrabackup_release -DCMAKE_INSTALL_PREFIX=/usr -DMYSQL_UNIX_ADDR=/var/run/mysqld/mysqld.sock \
-	  -DWITH_SSL=system -DINSTALL_MYSQLTESTDIR=/usr/share/percona-xtrabackup-test -DINSTALL_MANDIR=/usr/share/man .
+	cmake -DBUILD_CONFIG=xtrabackup_release -DCMAKE_INSTALL_PREFIX=/opt/percona-xtrabackup/2. -DMYSQL_UNIX_ADDR=/var/run/mysqld/mysqld.sock \
```

Comment:
in general it should look like 
export _prefix= /opt/percona-xtrabackup/${xbMajor}.${xbMinor}, but it's not a bug